app/assets/stylesheets/vendor/devices.scss

```diff
@@ -0,0 +1,152 @@
+@mixin border-radius($radius) {
+  -webkit-border-radius: $radius;
+  border-radius: $radius;
+}
+
+.device {
+  display: inline-block;
+  position: relative;
+  -webkit-box-sizing: content-box;
+  -moz-box-sizing: content-box;
+  box-sizing: content-box;
+
+  .screen {
+    width: 100%;
+    position: relative;
+    height: 100%;
+    color: white;
+    z-index: 2;
+    text-align: center;
+    display: block;
+    @include border-radius(1px);
+    -webkit-box-shadow: 0 0 0 3px #111;
+    box-shadow: 0 0 0 3px #111;
+  }
+
+  .top-bar, .bottom-bar {
+    height: 3px;
+    background: black;
+    width: 100%;
+    display: block;
+  }
+
+  .middle-bar {
+    width: 3px;
+    height: 4px;
+    top: 0px;
+    left: 90px;
+    background: black;
+    position: absolute;
+  }
+
+
+  &.nexus5 {
+    padding: 50px 15px 50px 15px;
+    width: 320px;
+    height: 568px;
+    background: #1e1e1e;
```

Comment:
Color literals like `#1e1e1e` should only be used in variable declarations; they should be referred to via variable everywhere else.